package.json

```diff
@@ -83,10 +83,10 @@
     "@emotion/react": "^11.11.3",
     "@emotion/styled": "^11.11.0",
     "@mnajdova/enzyme-adapter-react-18": "^0.2.0",
-    "@mui/icons-material": "^5.15.5",
-    "@mui/material": "^5.15.5",
-    "@mui/monorepo": "https://github.com/mui/material-ui.git#master",
-    "@mui/utils": "^5.15.5",
+    "@mui/icons-material": "^5.15.7",
+    "@mui/material": "^5.15.7",
+    "@mui/monorepo": "https://github.com/mui/material-ui#e1231ee8c18ff93b5129cfb5ff7c7e30ad8743e3",
```

Comment:
We are going to [have to switch](https://github.com/mui/mui-x/pull/11875/commits/37b89a9b56644fdb7d5c1cf5c55c67326a50507a) to such approach for `pnpm` anyways.
Why not now? 🤔

Reply:
@LukasTy You mean that with `pnpm` we will have to set the commit hash to bump the monorepo?

Reply:
Not specifically bump, but stop `pnpm dedupe` from failing. 😉 
When you run `pnpm dedupe` with `#master`, it finds that there is a newer commit and tries applying it... 🤷

Reply:
mui-toolpad has also gone the same route: https://github.com/mui/mui-toolpad/blob/1de30d83479572246a4f6f5984636fc24678cc2e/package.json#L42

Reply:
Should we do the same on the `next` branch?

Reply:
I'm not sure there is any rush for it.
Hopefully `pnpm` migration won't take too long. 🙈